src/freadR.c

```diff
@@ -158,6 +159,13 @@ SEXP freadR(
   // here we use bool and rely on fread at R level to check these do not contain NA_LOGICAL
   args.stripWhite = LOGICAL(stripWhiteArg)[0];
   args.skipEmptyLines = LOGICAL(skipEmptyLinesArg)[0];
+  if (!isString(commentCharArg) || LENGTH(commentCharArg) != 1)
+    error(_("comment.char must be a single character vector of length 1"));  // # notranslate
```

Comment:
NB: `_()` is the translation macro, I guess maybe you meant `# nocov` since this is tested on the R side?

(personally, I don't see much point in a check like this that can never be reached)

Reply:
Fine for me, I've removed the unreachable code here.